### DISCUSSION TITLE
Install local packages as part of environment activation

### DISCUSSION THREAD

**Discussion by amirhosseindavoody** [prefix-dev/pixi#883] [General]
Hi Pixi team,
Thanks for the awesome work.

I am wondering if there is a way to install local python packages as part of environment definition.

Currently I can do this with using `activation` section of the pixi.toml file and pip install a local package. But I think this is hack. I know that I can build the local package and install it from a local conda-channel, but I am looking for something like the `pip install -e`.

**Comment by pavelzw** [comment]
Not yet. You can follow this issue to see when there will be support: https://github.com/prefix-dev/pixi/issues/524

**Comment by ruben-arts** [comment]
As we speak we're working on the design for that: https://github.com/prefix-dev/pixi/issues/879

This depends on the [`uv` refactor](https://github.com/prefix-dev/pixi/pull/863).
